faf-commons-api/src/main/java/com/faforever/commons/api/dto/ModerationReport.java

```diff
@@ -0,0 +1,31 @@
+package com.faforever.commons.api.dto;
+
+import java.util.Collection;
+import java.util.Set;
+
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Type("moderationReport")
+@Getter
+@Setter
+public class ModerationReport extends AbstractEntity {
+  private String reportDescription;
+  private ModerationReportStatus reportStatus = ModerationReportStatus.AWAITING;
```

Comment:
Do you really wanna set default I doubt that, if you wanna make a patch then you want it to be null needing it to set to null on very patch is painfull. I would rather not have a default